templates/includes/leftpanel.html

```diff
@@ -19,6 +19,7 @@ <h3>Technical</h3>
     <li><a class="menu" href="{% url 'warcalc' %}">Combat calculator</a></li>
     <li><a class="menu" href="{% url 'nations' %}">Nations</a></li>
     <li><a class="menu" href="https://github.com/longturn" target="_blank" rel="noopener noreferrer">Github</a></li>
+    <li><a class="menu" href="https://longturn.readthedocs.io/en/latest/index.html" target="_blank" rel="noopener noreferrer">Docs</a></li>
```

Comment:
Is the documentation real `Technical`? The bigger part is about playing the game and using the client. I'd move the documentation to `Longturn` (maybe between `Rules` and `Screenshots`?.